mha_helper/ssh_helper.py

```diff
@@ -69,10 +69,10 @@ def make_ssh_connection(self):
         # Load the host keys and set the default policy, later on we may do strict host key checking
         self._ssh_client.load_system_host_keys()
         self._ssh_client.set_missing_host_key_policy(paramiko.WarningPolicy())
-
+        
         # Handle additional options that are passed to ssh via the '-o' flag
         # Some of the common options that are ignored are 'PasswordAuthentication' and 'BatchMode'
-        for ssh_opt in options.additional_options:
+        for ssh_opt in options.additional_options or []:
```

Comment:
You cannot iterate through a None value, If no additional options are set set to empty array.